db/seeders/correspondence_type_seeder.rb

```diff
@@ -2,17 +2,23 @@ class CorrespondenceTypeSeeder
 
   def seed!
     puts "----Seeding Correspondence Types----"
-    CorrespondenceType.find_or_create_by! name: 'Freedom of information request',
-                                          abbreviation: 'FOI',
-                                          escalation_time_limit: 3,
-                                          internal_time_limit: 10,
-                                          external_time_limit: 20,
-                                          deadline_calculator_class: 'BusinessDays'
-    CorrespondenceType.find_or_create_by! name: 'Subject Access Request',
-                                          abbreviation: 'SAR',
-                                          escalation_time_limit: 0,
-                                          internal_time_limit: 10,
-                                          external_time_limit: 40,
-                                          deadline_calculator_class: 'CalendarDays'
+
+    rec = CorrespondenceType.find_by(name: 'Freedom of information request')
```

Comment:
We should use `abbreviation` here, instead of `name`, because the abbreviation is the linchpin that attaches these records to `Case` and are the least likely to change, or if they do, the we're the most likely to investigate it's impact fully and notice that this needs to change here.